### PR TITLE
Wt 439 safari bug

### PR DESCRIPTION
**Pull request by rebelchris**
### Describe what this PR does
- Safari has to open the window outside any async calls, this way the window is not considered a blocking popup

## Events

Did you introduce any new tracking events?
Don't forget to update the [Analytics Taxonomy sheet](https://docs.google.com/spreadsheets/d/18Lv7zXges9QfVX5VYL1a-Hyl0e1sQ3sLr0OK8YZWKXI/edit#gid=0)

Log the new/changed events below:

| Type   | event_name  | value |
|--------|-------------|-------|
| Change/New | event name  | extra: { ... } |

### On those affected packages:
- [ ] Have you done sanity checks in the webapp?
- [ ] Have you done sanity checks in the extension?
- [ ] Does this not break anything in companion?

### Did you test the modified components media queries?
- [ ] MobileL (420px)
- [ ] Tablet (656px)
- [ ] Laptop (1020px)

#### Did you test on actual mobile devices?
- [ ] iOS (Chrome and Safari)
- [ ] Android

WT-439 #done

### CODE REVIEW

File: packages/shared/src/components/auth/AuthOptions.tsx [hunk 1]
```diff
@@ -118,7 +123,6 @@ function AuthOptions({
           name: getNodeValue('traits.name', connected.ui.nodes),
           email: getNodeValue('traits.email', connected.ui.nodes),
           image: getNodeValue('traits.image', connected.ui.nodes),
-          flowId: connected.id,
```
Comment (sshanzel) [inline packages/shared/src/components/auth/AuthOptions.tsx:121]: I think this was not intended.

File: packages/shared/src/components/auth/AuthOptions.tsx [hunk 2]
```diff
@@ -225,7 +229,7 @@ function AuthOptions({
         <Tab label={Display.SignBack}>
           <AuthSignBack
             onRegister={() => setActiveDisplay(Display.Default)}
-            onProviderClick={onSocialRegistration}
+            onProviderClick={onProviderClick}
```
Comment (sshanzel) [inline packages/shared/src/components/auth/AuthOptions.tsx:232]: Not sure if this was also part of the debugging phase.

**Review by sshanzel — APPROVED** [review]
Whoa! Can't believe this is all for fixing the Safari issue 🤯 

Other than the comments I have nothing to stop this amazing fix! 🚢

File: packages/shared/src/components/auth/AuthOptions.tsx (hunk 2 above)
Comment (rebelchris) [reply packages/shared/src/components/auth/AuthOptions.tsx:232]: No this needs to exist for the fix on login to work (not even sure why it was a seperate action)

File: packages/shared/src/components/auth/AuthOptions.tsx (hunk 1 above)
Comment (rebelchris) [reply packages/shared/src/components/auth/AuthOptions.tsx:121]: This should just be the same as the main branch, not sure why there was some change in my commit history

**Pull request merged**